presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataReader.java

```diff
@@ -299,10 +299,10 @@ private static Stream toStream(DwrfProto.Stream stream)
     {
         return new Stream(
                 stream.getColumn(),
+                stream.getSequence(),
```

Comment:
why did we change the order?

Reply:
sequence was unused and placed last along with other optional parameters. Now that we use it, I felt its more readable to have with along with column, as <column, sequence> identify the stream.